pkg/ddl/backfilling_scheduler.go

```diff
@@ -148,12 +148,6 @@ func initSessCtx(
 	sqlMode mysql.SQLMode,
 	tzLocation *model.TimeZoneLocation,
 ) error {
-	// Unify the TimeZone settings in newContext.
-	if sessCtx.GetSessionVars().StmtCtx.TimeZone() == nil {
-		tz := *time.UTC
-		sessCtx.GetSessionVars().StmtCtx.SetTimeZone(&tz)
-	}
-	sessCtx.GetSessionVars().StmtCtx.IsDDLJobInQueue = true
```

Comment:
Is this change expected? Though it seems that it's only used in `pkg/executor/ddl.go` 🤔 .

Reply:
I think `IsDDLJobInQueue` is used to make `ShouldClipToZeror` in the previous code. The verify CI passed after removing it.. @tangenta PTAL

Reply:
IIUC, `pkg/executor/ddl.go` is not related with `pkg/ddl/backfilling_scheduler.go` 🤔 , so it seems fine for this case.